app/models/family.rb

```diff
@@ -1,19 +1,21 @@
 class Family < ApplicationRecord
+  has_paper_trail
+
   include OrganizationScope
 
-  belongs_to :male, class_name: 'Animal'
-  belongs_to :female, class_name: 'Animal'
+  belongs_to :male, class_name: 'Animal', optional: true
```

Comment:
I've made these optional per the discussion this morning whereby you mentioned that a family could potentially not have a determined male or female in the beginning @benreyn is that okay?

Reply:
Yup. Good call. Under normal circumstances I probably would have done this in a separate PR, but fine to leave in here.